specification/translation/data-plane/Azure.AI.DocumentTranslation/stable/2024-05-01/openapi.json

```diff
@@ -0,0 +1,1463 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Returns the status for all documents in a batch document translation request": {
+            "$ref": "./examples/documents.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/document/batches/{id}/documents/{documentId}": {
+      "get": {
+        "operationId": "DocumentTranslationOperations_GetDocumentStatus",
+        "summary": "Returns the status for a specific document",
+        "description": "Returns the translation status for a specific document based on the request Id\nand document Id.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "description": "Format - uuid.  The batch id",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "documentId",
+            "in": "path",
+            "description": "Format - uuid.  The document id",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/DocumentStatus"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Returns the status of a single document in a batch translation request": {
+            "$ref": "./examples/document.json"
+          }
+        }
+      }
+    },
+    "/document/formats": {
+      "get": {
+        "operationId": "DocumentTranslationOperations_GetSupportedFormats",
+        "summary": "Returns a list of supported document formats",
+        "description": "The list of supported formats supported by the Document Translation\nservice.\nThe list includes the common file extension, as well as the\ncontent-type if using the upload API.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "description": "the type of format like document or glossary ",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "document",
+              "glossary"
+            ],
+            "x-ms-enum": {
```

Comment:
Please provide a description for each enum value, here and for other fields with an x-ms-enum. The Azure REST API docs generation will pick these up and display them in the public REST API doc for your service. For APIs authored in TypeSpec, you add descriptions to enum values with an `@doc` decorator on the enum values.

Reply:
Updated for all enum values